### PR TITLE
Migrate CI to Github actions

### PR DESCRIPTION
**Pull request by browniebroke**
https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

Result: https://github.com/festicket/django-better-choices/actions/runs/390077610

### CODE REVIEW

**Comment by lokhman** [comment]
Looks good. I will check the flow and come back to you. Thanks!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
```
Comment (lokhman) [inline .github/workflows/ci.yml:10]: This is a nice catch, as apparently, the default is `true` 👍

**Review by lokhman — APPROVED** [review]
Yes, Travis made things complicated. Actions configuration looks great. Happy to merge it 👍 

Thanks!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (browniebroke) [reply .github/workflows/ci.yml:10]: Yes I start to have some good experience with Github actions, this caught me a few times! 😄

**Pull request merged**